wiki/People/Users_with_unique_titles/en.md

```diff
@@ -0,0 +1,305 @@
+- ![][flag_ES] [Deif](https://osu.ppy.sh/users/318565)
+- ![][flag_MY] [ExPew](https://osu.ppy.sh/users/665612)
+- ![][flag_TW] [Firce777](https://osu.ppy.sh/users/274072)
+- ![][flag_JP] [Gamu](https://osu.ppy.sh/users/611174)
+- ![][flag_CA] [jonathanlfj](https://osu.ppy.sh/users/270377)
+- ![][flag_CA] [ktgster](https://osu.ppy.sh/users/53378)
+- ![][flag_NL] [Kyshiro](https://osu.ppy.sh/users/640611)
+- ![][flag_NL] [Lesjuh](https://osu.ppy.sh/users/44308)
+- ![][flag_TW] [Licnect](https://osu.ppy.sh/users/352804)
+- ![][flag_AU] [Lilac](https://osu.ppy.sh/users/58197)
+- ![][flag_ID] [LordRaika](https://osu.ppy.sh/users/3265023)
+- ![][flag_AU] [m980](https://osu.ppy.sh/users/3288)
+- ![][flag_US] [MMzz](https://osu.ppy.sh/users/128993)
+- ![][flag_CN] [Necho](https://osu.ppy.sh/users/4086593)
+- ![][flag_FR] [Nofool](https://osu.ppy.sh/users/672430)
+- ![][flag_US] [RLC](https://osu.ppy.sh/users/1047883)
+- ![][flag_MY] [RyoKazuka](https://osu.ppy.sh/users/6258586)
+- ![][flag_CN] [Ryuusei Aika](https://osu.ppy.sh/users/7777875)
+- ![][flag_SG] [Sinnoh](https://osu.ppy.sh/users/4236057)
+- ![][flag_LY] [Soul Evans](https://osu.ppy.sh/users/4490770)
+- ![][flag_KR] [Taeyang](https://osu.ppy.sh/users/2732340)
+- ![][flag_KR] [W h i t e](https://osu.ppy.sh/users/685229)
+- ![][flag_SE] [Xgor](https://osu.ppy.sh/users/98661)
+- ![][flag_CL] [ZiRoX](https://osu.ppy.sh/users/200768)
+
+### Aspire mapping contest winners
+
+The Aspire mapping contests featured a special variation of the Elite Mapper title called **Elite Mapper: Aspirant** as a prize.
+
+- ![][flag_US] [Charles445](https://osu.ppy.sh/users/85000)
+- ![][flag_CA] [Monstrata](https://osu.ppy.sh/users/2706438)
+- ![][flag_FI] [ProfessionalBox](https://osu.ppy.sh/users/3250792)
+- ![][flag_TW] [SnowNiNo_](https://osu.ppy.sh/users/2506267)
+- ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598)
+- ![][flag_FR] [Supairo](https://osu.ppy.sh/users/2837231)
+
+![][flag_US] [Charles445](https://osu.ppy.sh/users/85000) received this title after already having an Elite Mapper title, so it was "upgraded" to **Elite Mapper II: Aspirant**.
+
+### osu!remix contest winners
+
+The osu!remix contests award the **osu!mixer** title as a first-place prize.
+
+- ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471)
+
+### Featured Artists
+
+[Featured Artists](/wiki/Featured_Artists) with osu! accounts are given the **Featured Artist** title to show their involvement.
+
+- ![][flag_US] [antiPLUR](https://osu.ppy.sh/users/7318723)
+- ![][flag_AT] [creosu](https://osu.ppy.sh/users/13958834)
+- ![][flag_US] [Dictate](https://osu.ppy.sh/users/5983379)
+- ![][flag_SE] [Helblinde](https://osu.ppy.sh/users/48053)
+- ![][flag_DE] [KIRA](https://osu.ppy.sh/users/2138310)
+- ![][flag_JP] [nakanojojo](https://osu.ppy.sh/users/10631848)
+- ![][flag_US] [onumi](https://osu.ppy.sh/users/11204867)
+- ![][flag_ES] [sotuiofficial](https://osu.ppy.sh/users/14779258)
```

Comment:
Refine to say this is due to "Rising Sun Trax"?

Reply:
will put all of their fa pages here